content/docs/1.7.0/important-notes/_index.md

```diff
@@ -36,7 +39,7 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
 
 ### Environment Check Script
 
-Since Longhorn v1.7.0, we have introduced the [Longhorn Command-line Tool](../advanced-resources/longhornctl/). The functionality of the environment check script, [environment_check.sh](https://github.com/longhorn/longhorn/blob/master/scripts/environment_check.sh) overlaps with that of the Longhorn Command-line Tool. Therefore, the script has been deprecated in v1.7.0 and is scheduled for removal in v1.8.0.
+Since Longhorn v1.7.0, we have introduced the [Longhorn Command Line Tool](../advanced-resources/longhornctl/). The functionality of the environment check script, [environment_check.sh](https://github.com/longhorn/longhorn/blob/master/scripts/environment_check.sh) overlaps with that of the Longhorn Command Line Tool. Therefore, the script has been deprecated in v1.7.0 and is scheduled for removal in v1.8.0.
```

Comment:
```suggestion
The functionality of the [environment check script](https://github.com/longhorn/longhorn/blob/master/scripts/environment_check.sh) (`environment_check.sh`) overlaps with that of the Longhorn CLI, which is available starting with v1.7.0. Because of this, the script is deprecated in v1.7.0 and is scheduled for removal in v1.8.0.
```